Sources/Components/TextArea/TextAreaStyle.swift

```diff
@@ -0,0 +1,9 @@
+extension Warp {
+    /// Represents the different styles a `TextArea` can have.
+    public enum TextAreaStyle: String, CaseIterable {
+        case `default`
+        case disabled
+        case error
+        case readOnly
+    }
```

Comment:
Also, there is one more style, 'Active,' which shouldn't be accessible to the user, so I didn't define it here. In the Active style, we highlight the border, which is handled internally by the component.